### PR TITLE
FEATURE: BB-585 New Headline visualization 

### PR DESCRIPTION
**Pull request by tomassatinsky**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/700/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/569/ : SUCCESS in 2m 02s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/37/ : SUCCESS in 19s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/701/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/184/ : SUCCESS in 2m 48s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/705/ : SUCCESS in 20s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/574/ : SUCCESS in 2m 24s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/42/ : SUCCESS in 1m 47s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/706/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/575/ : SUCCESS in 2m 10s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/43/ : SUCCESS in 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/707/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/576/ : SUCCESS in 59s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/44/ : SUCCESS in 2m 01s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/708/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/577/ : SUCCESS in 49s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/45/ : SUCCESS in 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/709/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/578/ : SUCCESS in 46s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/46/ : SUCCESS in 23s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/710/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/579/ : SUCCESS in 2m 44s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/47/ : SUCCESS in 27s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/714/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/583/ : SUCCESS in 1m 09s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/51/ : SUCCESS in 20s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/716/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/585/ : SUCCESS in 46s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/53/ : SUCCESS in 19s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/177/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/717/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/186/ : UNSTABLE in 2m 46s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/718/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/586/ : SUCCESS in 52s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/54/ : SUCCESS in 22s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/178/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/719/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/187/ : SUCCESS in 2m 44s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/722/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/589/ : SUCCESS in 2m 17s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/57/ : SUCCESS in 1m 30s

File: src/components/core/base/VisualizationLoadingHOC.tsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+export function visualizationLoadingHOC<T extends ICommonVisualizationProps>(
+    InnerComponent: React.ComponentClass<T & ILoadingInjectedProps>
+): React.ComponentClass<T> {
+
+    return class WrappedComponent
+        extends React.Component<T & ILoadingInjectedProps, IVisualizationLoadingState> {
+
+        public static defaultProps: Partial<T & ILoadingInjectedProps> = InnerComponent.defaultProps;
+
+        protected subject: ISubject<IHeadlineDataPromise>;
```
Comment (vojty) [inline src/components/core/base/VisualizationLoadingHOC.tsx]: `Promise<Execution.IExecutionResponses>` vs `IHeadlineDataPromise`

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/777/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/625/ : SUCCESS in 2m 11s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/96/ : FAILURE in 1m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/778/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/626/ : SUCCESS in 48s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/97/ : FAILURE in 1m 04s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/779/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/627/ : SUCCESS in 1m 56s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/98/ : FAILURE in 1m 20s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/780/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/628/ : SUCCESS in 49s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/99/ : FAILURE in 1m 15s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/781/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/629/ : SUCCESS in 2m 14s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/100/ : FAILURE in 1m 23s

**Comment by tomassatinsky** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/785/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/633/ : SUCCESS in 54s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/104/ : SUCCESS in 23s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/788/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/637/ : SUCCESS in 2m 20s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/108/ : SUCCESS in 32s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/194/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/789/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/208/ : UNSTABLE in 4m 01s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/791/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/639/ : SUCCESS in 48s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/110/ : SUCCESS in 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/792/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/640/ : SUCCESS in 49s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/111/ : SUCCESS in 25s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/795/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/643/ : SUCCESS in 1m 54s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/114/ : SUCCESS in 23s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/195/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/797/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/209/ : UNSTABLE in 4m 08s

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/799/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/647/ : FAILURE in 1m 40s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/118/ : SUCCESS in 21s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/800/ : SUCCESS in 27s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/648/ : SUCCESS in 56s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/119/ : SUCCESS in 27s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/197/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/801/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/210/ : SUCCESS in 4m 01s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-merge-compliance-check/168/ : SUCCESS in 1m 05s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/649/ : SUCCESS in 56s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/198/ : SUCCESS in 3m 25s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/120/ : SUCCESS in 1m 42s

**Pull request merged**